google/cloud/bigquery/table.py

```diff
@@ -1274,6 +1288,30 @@ def __init__(self, resource):
             )
 
 
+class SnapshotDefinition:
+    """Information about base table and snapshot time of the snapshot.
+
+    See https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#snapshotdefinition
+
+    Args:
+        resource: Snapshot definition representation returned from the API.
+    """
+
+    def __init__(self, resource: Dict[str, Any]):
+        self.base_table_reference = None
+        if "baseTableReference" in resource:
+            self.base_table_reference = TableReference.from_api_repr(
+                resource["baseTableReference"]
+            )
+
+        self.snapshot_time = None
+        if "snapshotTime" in resource:
+            dt = google.cloud._helpers._rfc3339_to_datetime(resource["snapshotTime"])
+            # The helper returns a timezone *aware* datetime object (with UTC tzinfo),
+            # we need to make it naive.
```

Comment:
Why would we prefer a naive time here?  If the back-end is returning a Zulu-point-in-time, what benefit is there in discarding that information?

Reply:
If a timezone is not explicitly specified, the default time zone, UTC, is used for timestamps. But yeah, that's implicit, I'll convert it back to a tz-aware object.